### PR TITLE
readme: Update build instructions

### PR DESCRIPTION
**Pull request by Arxcis**
## Why?
- As a novice Flatseal-developer, I found the current build-instructions could be improved/updated.

### CODE REVIEW

**Review by tchx84 — CHANGES_REQUESTED** [review]
These steps are clearly better, but I don't think need all the extra sections? How about just compact all these into the same code block? Similarly to what we have now? But just with the extra steps for more clarity?

**Comment by Arxcis** [comment]
@tchx84 Good suggestion :+1: Updated in f1c3ea8. See updated screenshot above and actual render here -> https://github.com/Arxcis/Flatseal/tree/readme/update-build-instructions#build-it-yourself

File: README.md [hunk 1]
```diff
@@ -16,15 +16,32 @@ For more details please visit the [documentation](./DOCUMENTATION.md) page.
...
 
+Download the source code
 ```
-$ git clone https://github.com/tchx84/Flatseal.git
-$ cd Flatseal
-$ flatpak-builder --force-clean --repo=repo build com.github.tchx84.Flatseal.json
-$ flatpak build-bundle repo flatseal.flatpak com.github.tchx84.Flatseal
-$ flatpak install flatseal.flatpak
+git clone https://github.com/tchx84/Flatseal.git
+cd Flatseal
```
Comment (tchx84) [inline README.md:21]: What I commented before was to keep this in the same code block as the "build with flatpak" instructions.

File: README.md [hunk 2]
```diff
@@ -16,15 +16,32 @@ For more details please visit the [documentation](./DOCUMENTATION.md) page.
...
+flatpak-builder --user --install build com.github.tchx84.Flatseal.json
+flatpak run --branch=master com.github.tchx84.Flatseal
+```
+
+
+**Build it with Builder**
+
+```
+flatpak install org.gnome.Builder
+flatpak run org.gnome.Builder
```
Comment (tchx84) [inline README.md]: I think these instructions are out of scope for this README, please keep only the "build with flatpak" steps.

File: README.md [hunk 3]
```diff
@@ -16,15 +16,32 @@ For more details please visit the [documentation](./DOCUMENTATION.md) page.
...
-$ flatpak-builder --force-clean --repo=repo build com.github.tchx84.Flatseal.json
-$ flatpak build-bundle repo flatseal.flatpak com.github.tchx84.Flatseal
-$ flatpak install flatseal.flatpak
+git clone https://github.com/tchx84/Flatseal.git
+cd Flatseal
 ```
 
-Or just use [Builder](https://flathub.org/apps/details/org.gnome.Builder)
+
+**Build it with Flatpak**
```
Comment (tchx84) [inline README.md]: Since we're only keeping the "build with flatpak" steps, no need for the bold typed text.

**Review by tchx84 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (Arxcis) [reply README.md]: Ok. Fixed in 57f0edf

File: README.md (hunk 3 above)
Comment (Arxcis) [reply README.md]: Fixed in 57f0edf

File: README.md (hunk 1 above)
Comment (Arxcis) [reply README.md:21]: OK. Fixed in 57f0edf

**Comment by tchx84** [comment]
Thanks :)
